admin/static/coffee/modals.coffee

```diff
@@ -92,7 +92,7 @@ class DeleteDatabaseModal extends ui_modals.AbstractModal
 
 
     on_submit: =>
-        if @$('.verification_name').val() isnt @database_to_delete.get('name')
+        if @$('.verification_name').val().replace(/^\s+|\s+$/g,'') isnt @database_to_delete.get('name')
```

Comment:
Currently we use a lot that isn't IE8 compatible, but we try to polyfill where appropriate. For now I'd say leave it since it's just a code-clarity issue and it's not adding tons of boilerplate. I'll open an issue for making an official policy on which browsers we want the webui to support (including adding polyfills if necessary)